substrate/frame/election-provider-multi-phase/src/unsigned.rs

```diff
@@ -386,7 +386,7 @@ impl<T: Config> Pallet<T> {
 		ensure!(
 			Self::queued_solution().map_or(true, |q: ReadySolution<_, _>| raw_solution
 				.score
-				.strict_threshold_better(q.score, T::BetterUnsignedThreshold::get())),
+				.strict_threshold_better(q.score, sp_runtime::Perbill::zero())),
```

Comment:
nit: can this be a regular `cmp` since we have an `Ord` implementation for `ElectionScore`?

Reply:
Good point. Fixed now and added more test scenarios to check around the bounds.